packages/console/src/components/PlanGate.tsx

```diff
@@ -20,10 +21,16 @@ const PricingTable = ({
   const { isIframe } = useIframe()
   
   return (
-    <div className="flex flex-col justify-center items-center min-h-screen">
+    <div className="flex flex-col justify-center items-center min-h-screen relative">
       <div className="my-6">
         <Logo />
       </div>
+      <Link
+        href="https://storacha.console.network/logout"
+        className="absolute top-4 right-4 bg-hot-red text-white px-4 py-2 rounded-full shadow-md hover:border-white transition"
+      >
+        &larr; Back
```

Comment:
"Back" doesn't feel like it gives the user enough context here - I think it would be better to have this button say "Nevermind, Log Me Out" or something along those lines - I get that the only sensible thing to do if they want to leave this page is to log out, but "Back" makes it sound like we're going to do something else and I want to be super clear with the user about what they're doing.